src/Authentication/DefaultWebauthn2fAuthenticationChecker.php

```diff
@@ -19,7 +19,7 @@
  *
  * @package CakeDC\Auth\Auth
  */
-class DefaultWebauthn2fAuthenticationChecker implements Webauthn2fAuthenticationCheckerInterface
+class DefaultWebauthn2fAuthenticationChecker implements Webauthn2FAuthenticationCheckerInterface
```

Comment:
we should rename this file to DefaultWebauthn2FAuthenticationChecker (capital F) to match the name of the interface